app/src/components/governance/ParticipantList.vue

```diff
@@ -76,6 +76,10 @@ export default {
       type: Array,
       required: true,
     },
+    showAmounts: {
+      type: Boolean,
+      default: false
+    }
```

Comment:
I am just seeing in the `ParticipantList` code that the usage of `showAmounts` in quite counter-intuitive no?

Like it is exactly the opposite of what you would expect:

```
        <div v-if="!showAmounts && participant.option">
          <span class="option">{{ participant.option }}</span>
        </div>
        <div v-if="!showAmounts && participant.amount">
          <span class="amount">{{ participant.amount.amount }}</span>
          <span>{{ participant.amount.denom.concat(`s`) }}</span>
        </div>
```

We should definitely invert the values here

Reply:
yeah - shouldn't it be:

`<div v-if="showAmounts && participant.amount">`

Reply:
OK, I am going to change it in this other PR #4939

Reply:
`showAmounts` was there before and I think it referred to the amount of total stake?

Reply:
it refers whether or not the participants should display vote options & deposit amount. Yes, it was there and wrong from before